man/man5/zfs-module-parameters.5

```diff
@@ -638,6 +639,18 @@ Default value: \fB0\fR.
 .sp
 .ne 2
 .na
+\fBzfs_arc_min_long_lifespan\fR (int)
+.ad
+.RS 12n
+Minimum time "long-lived" blocks are locked in the ARC, specified in jiffies.
+A value of 0 will default to 6 seconds.
```

Comment:
Brian and I talked about this. I think we decided it would be best to make a pass through the code and cleanup all tunables that use jiffies in a later PR. This one simply works the same as `zfs_arc_min_prefetch_lifespan` for the moment.